sw/device/tests/chip_power_idle_load.c

```diff
@@ -0,0 +1,516 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//==============================================================================
+// Included code:
+
+#include <stdint.h>
+
+#include "sw/device/lib/base/mmio.h"  // Memory-mapped IO functions
+#include "sw/device/lib/dif/dif_alert_handler.h"
+#include "sw/device/lib/dif/dif_gpio.h"
+#include "sw/device/lib/dif/dif_otp_ctrl.h"
+#include "sw/device/lib/dif/dif_pinmux.h"
+#include "sw/device/lib/dif/dif_pwm.h"
+#include "sw/device/lib/dif/dif_pwrmgr.h"
+#include "sw/device/lib/dif/dif_rv_core_ibex.h"
+#include "sw/device/lib/dif/dif_rv_timer.h"
+#include "sw/device/lib/runtime/hart.h"  // functions for excution halt-like functionality
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/alert_handler_testutils.h"
+#include "sw/device/lib/testing/aon_timer_testutils.h"
+#include "sw/device/lib/testing/pwrmgr_testutils.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+#include "sw/device/lib/testing/test_framework/ottf_isrs.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+
+#include "alert_handler_regs.h"  // Generated.
+#include "aon_timer_regs.h"
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"  // Top-specific Definitions
+#include "pwm_regs.h"
+
+//==============================================================================
+// Global Variables
+
```

Comment:
Remove these type of block comments. They are just use for guidance in the test templates.

Reply:
While you are at it, maybe remove them throughout the PR, in other files too, thanks! It will be great to keep the codebase consistent.

Reply:
Done.